plugin/src/main/groovy/gorm/tools/repository/GormRepo.groovy

```diff
@@ -164,7 +164,7 @@ trait GormRepo<D extends GormEntity> implements GormBatchRepo<D>, MangoQueryTrai
     @Override
     void bind(Map args = [:], D entity, Map data, BindAction bindAction) {
         getRepoEventPublisher().doBeforeBind(this, entity, data, bindAction)
-        doBind(args, entity, data)
+        doBind(args, entity, data, bindAction)
```

Comment:
BindAction is already part of the args as set above. Does it need to be passed in a separate param?

Reply:
I see.